ozaria/site/components/teacher-dashboard/common/SecondaryTeacherNavigation.vue

```diff
@@ -346,6 +350,17 @@ export default {
       </router-link>
     </li>
 
+    <li v-if="isCodeCombat">
+      <a
+        id="OzariaAnchor"
+        :href="ozariaBaseURL"
+        data-action="Ozaria HomePage: Nav Clicked"
+        @click.native="trackEvent"
+      >
+        <div id="IconOzaria" />
+        <span>{{ $t('new_home.try_ozaria') }}</span>
```

Comment:
just call it Ozaria instead of Try Ozaria.

Also, open the link in new tab and move its position below HackStack in the sidebar